.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "github.copilot-nightly"
+    ]
+}
```

Comment:
don't need this file, can add `.vscode` to `.gitignore`

Reply:
Done

Reply:
did it wrong, just need to add: `.vscode` to file, not whole object... plus need to delete `.vscode` after it is stuck in Git now

Reply:
fixed it